commons/src/main/java/org/acme/model/Vulnerability.java

```diff
@@ -54,66 +72,109 @@ public enum Source {
         OSV // Google OSV Advisories
     }
 
-    private long id;
+    @Id
+    @Column(name = "ID")
+    private int id;
 
+    @Column(name = "VULNID", nullable = false)
+    @JsonDeserialize(using = TrimmedStringDeserializer.class)
     private String vulnId;
 
+    @Column(name = "SOURCE", nullable = false)
+    @JsonDeserialize(using = TrimmedStringDeserializer.class)
     private String source;
 
+    @Column(name = "FRIENDLYVULNID")
+    @JsonDeserialize(using = TrimmedStringDeserializer.class)
     private String friendlyVulnId;
 
+    @Column(name = "TITLE")
+    @JsonDeserialize(using = TrimmedStringDeserializer.class)
     private String title;
 
+    @Column(name = "SUBTITLE")
+    @JsonDeserialize(using = TrimmedStringDeserializer.class)
     private String subTitle;
 
+    @Column(name = "DESCRIPTION")
+    @JsonDeserialize(using = TrimmedStringDeserializer.class)
     private String description;
 
+    @Column(name = "DETAIL")
+    @JsonDeserialize(using = TrimmedStringDeserializer.class)
     private String detail;
 
+    @Column(name = "RECOMMENDATION")
+    @JsonDeserialize(using = TrimmedStringDeserializer.class)
     private String recommendation;
 
+    @Column(name = "\"REFERENCES\"")
```

Comment:
You shouldn't have to use quotes here when `quarkus.hibernate-orm.database.globally-quoted-identifiers=true` is set. 

See:

https://github.com/mehab/DTKafkaPOC/blob/3eab7f96312241a13990e4cffebf0c2b2bdd6509/vulnerability-analyzer/src/main/resources/application.properties#L61-L64

Reply:
REFERENCES is sql keyword, so naming it without quotes was throwing error